pkgs/development/python-modules/selenium/default.nix

```diff
@@ -31,6 +32,11 @@ buildPythonPackage rec {
     cd py
   '';
 
+  postInstall = ''
+    install -Dm 755 ../rb/lib/selenium/webdriver/atoms/getAttribute.js $out/${python.sitePackages}/selenium/webdriver/remote/getAttribute.js
```

Comment:
Might be worth adding the ref to upstream issue (<https://github.com/SeleniumHQ/selenium/issues/9917>) as part of the changeset's comment so that we can easily check (using blame) when this workaround is no longer required.

Reply:
It might be a good idea to create a separate PR for the selenium change as it as more chances to get merged faster.

Reply:
> Might be worth adding the ref to upstream issue ([SeleniumHQ/selenium#9917](https://github.com/SeleniumHQ/selenium/issues/9917)) as part of the changeset's comment so that we can easily check (using blame) when this workaround is no longer required.

The real problem is that we can't easily run their bazel build infrastructure and I fail to understand what needs to be run to only install those files. 

> as part of the changeset's comment

I am not sure what you mean by that. You want to add it to this commit?